README.md

```diff
@@ -23,8 +23,13 @@ npm install -g yo
 # install the react-server generator
 npm install -g generator-react-server
 
-# make a new react-server project
+# make a new react-server project in the CURRENT directory
```

Comment:
FWIW, I just tried `yo generator-webapp`, which is the example on Yeoman's README, and it drops all its files in the current directory, so I guess that's the standard. Interestingly, they don't tell you to make a sub-directory in their example.